en/user_guide/content.rst

```diff
@@ -0,0 +1,61 @@
+Contents
+========
+
+Contens are instances of a Content Types. A Content Type is the pattern of a family of Contents. If
```

Comment:
I would rather either :
 - put content and content type on plural
 - put content and content type on singular